onnxruntime/python/tools/kernel_explorer/kernel_explorer.cc

```diff
@@ -39,6 +39,14 @@ KE_REGISTER(m) {
     return true;
 #else
         return false;
+#endif
+  });
+
+  m.def("is_hipblaslt_available", []() {
+#ifdef USE_HIPBLASLT
+    return true;
+#else
+        return false;
```

Comment:
Is such identation as expected? Similar to line 41.

Reply:
we need identation to pass CI.